### PR TITLE
fixing setup import of requirements

### PR DESCRIPTION
**Pull request by bryan-woods**
pip install is broken because of a change to how requirements are calculate in pip10

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,5 +1,8 @@
 from setuptools import setup
-from pip.req import parse_requirements
+try: # for pip >= 10
+    from pip._internal.req import parse_requirements
```
Comment (mcgibbon) [inline setup.py]: It's improper to import from a private module (one that starts with an underscore). Can you solve the issue without doing this?

**Review by mcgibbon — CHANGES_REQUESTED** [review]
We need a fix that won't break if pip decides to change its internal structure. I may also fix other issues that prevent Travis from running before I merge. Thanks for catching this, I look forward to a fix we can merge!

**Comment by bryan-woods** [comment]
Have you thought about using conda for the travis job?
https://conda.io/docs/user-guide/tasks/use-conda-with-travis-ci.html

**Comment by mcgibbon** [comment]
I have thought about it, but failed to get it to work in practice. If you get it to work I'd merge it.

**Pull request closed without merging**